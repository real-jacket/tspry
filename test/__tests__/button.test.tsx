import React from 'react'
import render from 'react-test-renderer'
import Button from '../../lib/button'

describe('button', () => {
	it('is a div', () => {
		const json = render.create(<Button />).toJSON()
		expect(json).toMatchSnapshot()
	})
})
