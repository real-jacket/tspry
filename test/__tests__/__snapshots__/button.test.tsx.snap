// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button is a div 1`] = `
<div>
  button
</div>
`;
